test/widgets/lightbox_test.dart

```diff
@@ -109,6 +189,16 @@ void main() {
   TestZulipBinding.ensureInitialized();
 
   group("VideoLightboxPage", () {
+    void verifySliderPosition(WidgetTester tester, Duration duration) {
+      final slider = tester.widget(find.byType(Slider)) as Slider;
+      check(slider.value)
+        .equals(duration.inMilliseconds.toDouble());
+      final positionLabel = tester.widget(
+        find.byKey(VideoPositionSliderControl.kCurrentPositionLabelKey)) as VideoDurationLabel;
+      check(positionLabel.duration)
+        .equals(duration);
```

Comment:
This checks that the `VideoDurationLabel` widget gets the intended `Duration` value, but we don't yet check that `VideoDurationLabel` correctly renders that `Duration` into the `hh:mm:ss` format. That would be good to do, either here or in some separate unit tests.